graphite/graphite.go

```diff
@@ -42,9 +43,22 @@ func (r *Request) Query(host string) (Response, error) {
 	if r.End != nil {
 		v.Add("until", fmt.Sprint(r.End.Unix()))
 	}
+	u1, err := url.Parse(host)
+	if err != nil {
+		log.Printf("There was a problem parsing dev.conf: graphiteHost=" + host)
+		return nil, err
+	}
+	log.Printf("The scheme is : " + u1.Scheme + " and the host is : " + u1.Host)
+	if u1.Host == "" && u1.Scheme != "" {
```

Comment:
That would mean that we don't anticipate the way somebody will enter it. 
In essence here @mjibson , we check all the possible formats:
1)graphiteHost = <hostname> 
In this case u1.Scheme and u1.Host is empty because it parses them , looking for ':' and breaks.
2)graphiteHost = <proto><hostname>
In this case u1.Scheme and u1.Host is <proto> and <hostname> respectively. Which is fine
3)graphiteHost = <hostname>:<port>
In this case u1.Scheme is <hostname> and u1.Host is empty. 
It's alla about the url.Parse which is not that good yet. 
Hence the control flow , that's why I'd like to keep it . 
MM